server/server/src/sync/sync_api_v5.rs

```diff
@@ -22,13 +23,15 @@ pub enum RemoteSyncActionV5 {
 
 #[derive(Clone, Debug, Deserialize, Serialize)]
 pub struct RemoteSyncRecordV5 {
-    #[serde(rename = "syncId")]
+    #[serde(rename = "syncOutId")]
```

Comment:
mSupply was rejecting the remote sync, as it expects `syncOutId` not `syncId`
Ditto with the other fields below..